packages/core-typings/src/IMessage/MessageAttachment/Files/FileProp.ts

```diff
@@ -1,7 +1,7 @@
 export type FileProp = {
 	_id: string;
-	name: string;
-	type: string;
-	format: string;
-	size: number;
+	name?: string;
+	type?: string;
+	format?: string;
+	size?: number;
```

Comment:
Hmmm, a type with only ID as required, maybe it's not the proper type for your use case?